packages/angular_devkit/core/src/utils/strings.ts

```diff
@@ -89,7 +90,7 @@ export function classify(str: string): string {
   return str
     .split('.')
     .map((part) => capitalize(camelize(part)))
-    .join('.');
+    .join('');
```

Comment:
I'm wondering if this would be a breaking change since it is public and it could be expected to operate in the previous manner.

Reply:
Umm… not sure show to classify this. Maybe we should discuss it during our next sync.